### PR TITLE
Mika

### PR DESCRIPTION
**Pull request by Tanosimboangy**
Hello my dear ,

This is my  task ,could you verify please in case something needed to change.

Thanks in advance !

Jacquit

### CODE REVIEW

File: m/mika.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Mika sy Davis
+
+![alt text](MIKA.jpg)
```
Comment (ndorfin) [inline m/mika.md]: Please make the image filename match the artist filename.

Comment (ndorfin) [reply m/mika.md]: And replace the `alt text` with some text describing the image

File: m/mika.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+This artist started in 2005 and became famous in 2006
+
+**Genre:** Basesa
+
+**Artist homepage:** [MIKA & DAVIS](https://web.facebook.com/mikasydavis/?_rdc=1&_rdr) Mika sy Davis is a folk-fusion group from Madagascar composed of two male singer-songwriters named Mika and Franklin Davis. They perform a contemporary form of the traditional basesa genre of the island's east coast,[1] blended with rock, soul and reggae.[2] The band has been credited with revitalizing and nationally popularizing the basesa genre.[1]
```
Comment (ndorfin) [inline m/mika.md]: The bio you entered here should replace the 'This artist started...' bit above.
Make sure to tidy it up so it's readable, all the `[1]`s and `[2]`s need to be removed.

**Review by ndorfin — CHANGES_REQUESTED** [review]

File: m/mika.md (hunk 2 above)
Comment (Tanosimboangy) [reply m/mika.md]: I've made the changes that are needed to fix.could you check it please

File: m/mika.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Mika sy Davis
+
+![Image of mika & davis](mika.jpg)
```
Comment (ndorfin) [inline m/mika.md]: this image filename needs to match the filename of the artist.
There's already an artist called Mika, we don't want to create confusion!

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Review by ndorfin — CHANGES_REQUESTED** [review]
Please remove the `[1]` etc from the bio

File: m/mika.md (hunk 3 above)
Comment (Tanosimboangy) [reply m/mika.md]: I've finished it now, so please could you check it !

**Review by ndorfin — APPROVED** [review]
Jacquit, this is perfect now. Thanks! 💯

**Pull request merged**